src/apps/AppManager.jsx

```diff
@@ -1,278 +1,30 @@
-import { CompassApp } from '../containers/CompassApp';
+import {
+    CompassApp
+} from '../containers/CompassApp';
 import FullApp from '../containers/FullApp';
 import SlimApp from '../containers/SlimApp';
 import SmartApp from '../containers/SmartApp';
+import LandingPage from '../containers/SmartApp';
```

Comment:
I'm assuming this was unintentional -- LandingPage should be LandingPage

Reply:
Very much so, will address